### PR TITLE
Bedtools pipe transformer docs and notebook

### PR DESCRIPTION
**Pull request by williambrandler**
## What changes are proposed in this pull request?
added notebook from here
https://field-eng.cloud.databricks.com/#notebook/1344743

## How is this patch tested?
- [ ] Unit tests
- [ ] Integration tests
- [ ] Manual tests

(Details)

### CODE REVIEW

File: docs/source/etl/utility-functions.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Glow includes a variety of utility functions for performing basic data manipulat
 Struct transformations
 ======================
 
-Glow's struct transformation functions change the schema of a struct. These transformations integrate with functions
+Glow's struct transformation functions changes the schema structure of the DataFrame. These transformations integrate with functions
```
Comment (karenfeng) [inline docs/source/etl/utility-functions.rst]: `changes` -> `change`; there are multiple functions.

File: docs/source/tertiary/pipe-transformer.rst [hunk 2]
```diff
@@ -68,19 +70,16 @@ partition of the input DataFrame as VCF by choosing ``vcf`` as the input and out
...
-.. invisible-code-block: python
-
-   assert [f for f in info_free_df.schema.fieldNames() if f.startswith('INFO_')] == []
 
-When you use the VCF input formatter, you must specify a method to determine the VCF header. The
-simplest option is ``infer``, which instructs the Pipe Transformer to derive a VCF header from the
-DataFrame schema.
+When you use the VCF input formatter, you must specify a method to determine the VCF header. The option ``infer``
+instructs the Pipe Transformer to derive a VCF header from the DataFrame schema. Or you can provide the header
+as a blob, or you can point to the filesystem path for an existing VCF file with the correct header.
```
[the remark this answers is not in the document]
Comment (williambrandler) [reply docs/source/tertiary/pipe-transformer.rst]: although verbose, it was the fact that it was unexplained here that confused me originally

File: docs/source/etl/utility-functions.rst (hunk 1 above)
Comment (williambrandler) [reply docs/source/etl/utility-functions.rst]: oh yeah

File: docs/source/tertiary/pipe-transformer.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (karenfeng) [reply docs/source/tertiary/pipe-transformer.rst]: Sure. I'll keep the text here and add a link to the table as well so they know where to go.

**Comment by karenfeng** [comment]
Thanks for the contribution @williambrandler! Merging.

**Pull request merged**